### ISSUE TITLE
Namespace

### ISSUE THREAD

**Issue by MarcAntoine-Arnaud** [RReverser/serde-xml-rs#50]
Do you have any way to get namespace during deserialisation ?
And to add prefixes (and definition) of them for serialisation ?

**Comment by or29544** [comment]
Yes, I want to make an RSS parser and it also has namespaces. Especially when the RSS feed wants to be smart and implement both atom and rss standards in the same output...Right now serde-xml-rs will just abort deserialization complaining there are two identical fields in the XML. Yes there are, but one of them is namespaced.

**Comment by MarcAntoine-Arnaud** [comment]
Old issue !! héhé
I have created a project named [Yaserde](https://github.com/media-io/yaserde) to handle all specific features of XML.
It can be possible to use Yaserde with Serde, so like that an XML can be wroten in Json etc.

It provides today a pretty good coverage of XML features.

**Comment by Cabbache** [comment]
If it helps, I've modified the parser so that it includes the namespace name and colon in the attribute name: https://github.com/RReverser/serde-xml-rs/pull/216
